### ISSUE TITLE
notification not working on IOS8

### ISSUE THREAD

**Issue by qualitybits** [katzer/cordova-plugin-local-notifications#350]
notification is working perfectly on the IOS7(iPhone 4) but not working on IOS8 (iPad).

**Comment by katzer** [comment]
They canceled all pending submissions before turning PGB into a payed service (even for plugins and not only for apps!)

**Comment by psimondk** [comment]
Hi Katzer,

Great plugin - I changed as suggested and it works like a charm. Issue closed as far as I am concerned :+1: 

I don't get the stance with PGB. I used it originally but caved in and bought a macbook for IOS deployment. So although 'manual' Android deployment is super easy and 'manual' IOS deployment is super frustrating I will  need to see some real PGB improvements in order to re-consider.
